components/forms/Toggle/style.module.scss

```diff
@@ -0,0 +1,68 @@
+@import "styles/index.scss";
+
+.toggle {
+  $size: 14px;
+
+  align-items: center;
+  align-content: center;
+  position: relative;
+  height: $size;
+  width: $size;
+  flex-shrink: 0;
+
+  &__switch {
+    position: relative;
+    display: inline-block;
+  }
+
+  &__input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+
+  &__slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: $color-gray;
+    -webkit-transition: 0.4s;
```

Comment:
The code runs through autoprefixer, so no prefixes are needed. Please remove all -webkit- and other vendor prefixes.